### PR TITLE
7guis examples

### PR DESCRIPTION
**Pull request by rcqls**
The goal is to propose a mostly complete version of 7guis examples.

### CODE REVIEW

**Comment by dumblob** [comment]
Just a heads up: in the mean time a 7guis organization has been put together so the whole project is alive again: https://github.com/7guis/7guis/ .

Feel free to make a pull request there once you think it's publishable :wink:. Thanks!

**Comment by rcqls** [comment]
> Just a heads up: in the mean time a 7guis organization has been put together so the whole project is alive again: https://github.com/7guis/7guis/ .
> 
> Feel free to make a pull request there once you think it's publishable 😉. Thanks!

Yep! I had this in mind too but just after ending up the 7 examples. The last example 7gui-cells however would have  all the code inside v ui since integrated to the grid component.

**Comment by dumblob** [comment]
>The last example 7gui-cells however would have all the code inside v ui since integrated to the grid component.

You decide. I myself am currently not up to date with *V* development.

File: tool_gg.v [hunk 1]
```diff
@@ -58,3 +58,7 @@ pub fn (hs HexColor) color() gx.Color {
 	u := ('0x' + hs[1..]).u32()
 	return gx.rgba(u8(u >> 24), u8(u >> 16), u8(u >> 8), u8(u))
 }
+
+pub fn alpha_colored(c gx.Color, a u8) gx.Color {
+	return gx.rgba(c.r, c.g, c.b, a)
+}
```
Comment (spytheman) [inline tool_gg.v:64]: why is that needed? Calling `gx.rgba(r, g, b, a)` seems more intuitive to me 🤔, i.e. the Color already does have an alpha component.

Comment (rcqls) [reply tool_gg.v:64]: It is to obtain a alpha version of a fixed color like `ui.alpha_colored(gx.light_gray, 100)`.
